packages/backend/src/database/entities/organizations.ts

```diff
@@ -6,11 +6,15 @@ export type DbOrganization = {
     organization_name: string;
     allowed_email_domains: any; // jsonb
     created_at: Date;
+    chart_colors: string[];
```

Comment:
Yes, it is optional, I can set it to `string[] | undefined` but I don't see any impact in the API , do you know if there is any difference? 

If chart_colors is null I get this 

![image](https://user-images.githubusercontent.com/1983672/160845945-f2a8fed3-8a50-4ab7-883d-8a08589dbf5d.png)

Reply:
It's just that if `.chart_colors` can be null, then we should declare:

```typescript
chart_colors: string[] | undefined
```

That way we'll get a compile error if we do `chart_colors[0]` since `undefined[0]` would throw an error `Uncaught TypeError: Cannot read property '0' of undefined`

Reply:
is `chart_colors: string[] | undefined` the same as `chart_colors?: string[] `